mule-packager/src/test/java/org/mule/tools/api/MulePackageBuilderTest.java

```diff
@@ -1,217 +0,0 @@
-/*
- * Mule ESB Maven Tools
- * <p>
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * <p>
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-
-package org.mule.tools.api;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.mule.tools.api.packager.structure.FolderNames.CLASSES;
-import static org.mule.tools.api.packager.structure.FolderNames.MAVEN;
-import static org.mule.tools.api.packager.structure.FolderNames.META_INF;
-import static org.mule.tools.api.packager.structure.FolderNames.MULE_ARTIFACT;
-import static org.mule.tools.api.packager.structure.FolderNames.MULE_SRC;
-import static org.mule.tools.api.packager.structure.FolderNames.REPOSITORY;
-
-import java.io.File;
-import java.io.IOException;
-import java.lang.reflect.Field;
-import java.nio.file.Path;
-import java.util.List;
-
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.RunWith;
-import org.mockito.junit.MockitoJUnitRunner;
-
-import org.mule.tools.api.packager.archiver.MuleArchiver;
-import org.mule.tools.api.packager.builder.MulePackageBuilder;
-import org.mule.tools.api.packager.builder.PackageBuilder;
-import org.mule.tools.api.packager.packaging.PackagingOptions;
-import org.mule.tools.api.packager.packaging.PackagingType;
-
-@RunWith(MockitoJUnitRunner.class)
-public class MulePackageBuilderTest {
```

Comment:
what happened to this class?

Reply:
moved to the proper package as the class it was testing